changelog.txt

```diff
@@ -1,5 +1,22 @@
 *** Changelog ***
 
+2022-08-24 - version 4.6.3
+* New: Course List block (beta)
+    * Add the Course List block [#5419](https://github.com/Automattic/sensei/pull/5419)
+    * Add the Course Categories block [#5455](https://github.com/Automattic/sensei/pull/5455)
+    * Add new patterns for Course List [#5433](https://github.com/Automattic/sensei/pull/5433)
+    * Add Course Actions block [#5430](https://github.com/Automattic/sensei/pull/5430)
+    * Make Continue button take user to the lesson they were working on [#5496](https://github.com/Automattic/sensei/pull/5496)
+    * Make the Course List block with inner blocks globally available [#5473](https://github.com/Automattic/sensei/pull/5473)
+    * Add notice to show invalid usage when blocks are used out of course context [#5489](https://github.com/Automattic/sensei/pull/5489)
+* Add: Support for passing a custom footer to the Modal component [#5503](https://github.com/Automattic/sensei/pull/5503)
+* Fix: Template selection logic when learning mode is active [#5514](https://github.com/Automattic/sensei/pull/5514)
+* Fix: Course Outline - Show private lessons only for those who can view them. [#5468](https://github.com/Automattic/sensei/pull/5468)
+* Tweak: Use Ubuntu Latest instead of Ubuntu 18.04 on Github Actions workflow [#5506](https://github.com/Automattic/sensei/pull/5506)
```

Comment:
We can remove this one since it doesn't affect any of the files that actually ship to the user, so they won't care.

Reply:
Thanks @donnapep, removed [here](https://github.com/Automattic/sensei/pull/5535/commits/e1e33ef182b7ab77f15ef98b90ab3d16b807b358)